docs/api/app.md

```diff
@@ -852,6 +852,19 @@ correctly.
 
 **Note:** This will not affect `process.argv`.
 
+### `app.getColorizationColor()` _Windows_
+
+Returns the users current system wide color preference in the form of an ARGB
```

Comment:
Ya, that works too - I'm just not sure that people will know what this is. I'd even go so far as to rename this `getAccentColor`